src/mfpbench/tabular.py

```diff
@@ -168,18 +168,27 @@ def is_constant(_s: pd.Series) -> bool:
         start = sorted_fids[0]
         end = sorted_fids[-1]
         step = sorted_fids[1] - sorted_fids[0]
+        self._start = sorted_fids[0]
+        self._end = sorted_fids[-1]
+        self._step = sorted_fids[1] - sorted_fids[0]
```

Comment:
These are set in `benchmark` and obtained from the `self.fidelity_range`. I've made sure to drop the `0` epoch from lcbench tabular data such that:
```python
bench.start = 1
bench.end = 51
bench.step = 1
bench.fidelity_range = (1, 51, 1)
```